custom_components/dyson_local/manifest.json

```diff
@@ -6,7 +6,7 @@
     "issue_tracker": "https://github.com/shenxn/ha-dyson/issues",
     "dependencies": ["zeroconf"],
     "codeowners": ["@shenxn"],
-    "requirements": ["libdyson==0.8.9"],
-    "version": "0.16.3",
+    "requirements": ["libdyson @ git+https://github.com/shenxn/libdyson@main"],
```

Comment:
Could you please change this line to something like `git+https://github.com/shenxn/libdyson@main#egg=libdyson` right now it doesn't pass the checks

Reply:
I modified it as `libdyson@git+https://github.com/shenxn/libdyson@main` with a new commit below. Full disclosure, I have not had a chance to test and confirm this will work, but the original version worked fine.